.github/scripts/install_slices/install_slices.py

```diff
@@ -280,6 +293,49 @@ def install_slice(slice: str, arch: str, release: str) -> None:
                 res.stderr.rstrip(),
             )
             sys.exit(res.returncode)
+        # Check if the copyright file has been installed with this slice
+        if list(
+            filter(
+                lambda path: f"/usr/share/doc/{pkg}/copyright" in str(path)
+                and path.is_file(),
+                pathlib.Path(tmpfs).rglob("*"),
+            )
+        ):
+            missing_copyright[slice_name] = False
+        else:
+            missing_copyright[slice_name] = True
+
+
+def deb_has_copyright_file(pkg: str) -> bool:
+    """
+    Checks if a deb's contents comprise a copyright file
+
+    NOTE: this is a temporary and convoluted implementation, as at the moment
+    we don't have an easy and reliable way to check which deb was used for
+    the installation (at least not without duplicating a some of the Chisel
+    codebase).
+
+    TODO: update this function once the Chisel DB is available, as the pkg
+    SHAs will be available from the DB itself.
+    """
+    pkg_shas = os.popen(
+        f"grep -A 25 -R 'Package: {pkg}$' {str(CHISEL_PKG_CACHE)} | grep SHA256 | cut -d ' ' -f 2"
+    ).read()
+    for deb in pkg_shas.splitlines():
```

Comment:
If I understand this correctly, this might be problematic if we have different architectures and different versions of the same package. Maybe as a good-enough solution for now, we could remove the cache or start a new container for each architecture / version test (if we are not doing that already).

Reply:
That's not a problem because this is running in a matrix, with the `arch` and `release` as inputs. So you'll get an executor for every combination of those, and thus a clean worker/cache